### ISSUE TITLE
SUGGESTION: Implement a way to add a custom LUT

### ISSUE THREAD

**Issue by Zeioth** [benjamimgois/goverlay#142]
Right now we can add lut in available effects, but without configuring it, it won't do anything.

**Comment by benjamimgois** [comment]
Yeah, i know. To be honest, there's lot's of effects that doesn't work very well on vkbasalt. I think we need a whitelist of curated effects that we know it works well. I'll need some help with that testing.
